modules/nf-core/cellranger/count/templates/cellranger_count.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```
a blatantly nitpicky but paranoid suggestion here, based on personal experience

I still encounter systems and OSes with Python 2.7 as the default executable 👎🏼